src/redux/Books/Books.js

```diff
@@ -0,0 +1,26 @@
+// actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Good job so far :+1: Kindly use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your action type names. As a result, it should be something like this:
 ```npm
const ADD_BOOK = bookstore-react/books/ADD_BOOK;
const REMOVE_BOOK = bookstore-react/books/REMOVE_BOOK;
```